etcdutl/snapshot/v3_snapshot.go

```diff
@@ -265,6 +276,19 @@ func (s *v3Manager) Restore(cfg RestoreConfig) error {
 	if err = s.saveDB(); err != nil {
 		return err
 	}
+
+	if cfg.RevisionBump > 0 {
```

Comment:
I think `!= 0` would be more idiomatic. I'm not sure if we even want to allow to pass negative numbers in the first place?

Reply:
Yeah, I can't think of a reason that we would want to allow someone to decrease the revision, especially since the bump is there to help make sure that doesn't happen.